docs/markdown/snippets/mtest_exclude.md

```diff
@@ -0,0 +1,3 @@
+## `meson test` allows excluding individual tests
+
+The `test` subcommand now accepts `--exclude` argument, expecting the same same syntax as used by positional arguments.
```

Comment:
> expecting the same same syntax as used by positional arguments

That's an important detail that seems to be only mentioned in release notes. I think it should be said in other places too.

Reply:
“same same" looks like a typo?

Reply:
According to https://mesonbuild.com/Unit-tests.html#run-subsets-of-tests, it seems that the syntax of positional arguments is not exactly equal to the current `--exclude` implementation because the front one supports wildcard.

Reply:
Yes, https://github.com/mesonbuild/meson/commit/e7b9dfac98d30eb4ea5489c10b8dfef3bb8331f4 has been merged causing a conflict.

Unfortunately, if we want to support wildcards, we can no longer use the elegant set-based system.

Reply:
I think we can use the same pattern for excludes, with wildcard as well.

Reply:
From my point of view, I think not supporting wildcards is acceptable (but we should document it), it is always better than nothing, we can add wildcard functionality when needed.

Reply:
Just seems easier to reuse the same code.